core/src/services/alluxio/error.rs

```diff
@@ -46,9 +46,9 @@ pub async fn parse_error(resp: Response<IncomingAsyncBody>) -> Result<Error> {
 
     if let Some(alluxio_err) = alluxio_err {
         kind = match alluxio_err.status_code.as_str() {
-            "AlreadyExists" => ErrorKind::AlreadyExists,
-            "NotFound" => ErrorKind::NotFound,
-            "InvalidArgument" => ErrorKind::InvalidInput,
+            "ALREADY_EXISTS" => ErrorKind::AlreadyExists,
```

Comment:
Maybe we can add unit test to cover this?

Reply:
done